lib/src/data_frame/helpers/data_frame_to_string.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:ml_dataframe/ml_dataframe.dart';
+
+class _SeriesDisplayData {
+  _SeriesDisplayData(this.colIndex, this.colTitle) : maxChars = colTitle.length;
+  int colIndex = 0;
+  int maxChars;
+  String colTitle;
+  List<String> data = [];
+}
+
+const kPadding = 3;
+const kSkippingSymbol = '...';
+
+String dataFrameToString(DataFrame dataFrame, {maxRows = 10, maxCols = 7}) {
+  final rowCountStartEnd = [
+    (maxRows / 2).toInt(),
+    maxRows - (maxRows / 2).toInt()
+  ];
```

Comment:
@tadeohepperle could you please add a new line after this line (for better formatting)?